the command 'fish -c "uname"', and most other uses of fish's -c argument,
are not working if this is enabled.

--- a/build.rs
+++ b/build.rs
@@ -90,7 +90,7 @@ fn detect_cfgs(target: &mut Target) {
             target.has_symbol("localeconv_l")
         }),
         ("FISH_USE_POSIX_SPAWN", &|target| {
-            target.has_header("spawn.h")
+            false
         }),
         ("HAVE_PIPE2", &|target| {
             target.has_symbol("pipe2")
